extensions/sql-database-projects/src/models/project.ts

```diff
@@ -156,6 +156,8 @@ export class Project implements ISqlProject {
 		this.projFileXmlDoc = new xmldom.DOMParser().parseFromString(projFileText.toString());
 
 		await this.readProjectProperties();
+		await this.readSqlCmdVariables();
```

Comment:
I wasn't sure if it's better to set `this._sqlCmdVariables` in the helper function or have it return the result like before, so I went with setting it in the helper function like `readProjectProperties()`

Reply:
I like having the helper function set it better; I was already planning on making that conversion for the rest of them!